File: templates/template.py
import sys
import time
import tracemalloc
from typing import List, Tuple, Set, Dict

def solve():
    # Your solution here
    pass

def main():
    tracemalloc.start()
    start_time = time.time()
    
    tests = 1
    # tests = int(input())
    
    for _ in range(tests):
        solve()
    
    end_time = time.time()
    current, peak = tracemalloc.get_traced_memory()
    tracemalloc.stop()
    
    print("\n-------------------", file=sys.stderr)
    print(f"Time: {end_time - start_time:.6f} sec", file=sys.stderr)
    print(f"Memory: {peak / 1024 / 1024:.2f} MB", file=sys.stderr)

if __name__ == "__main__":
    main()

File: templates/template.java
import java.util.*;
import java.io.*;

public class Main {
    static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    static PrintWriter out = new PrintWriter(System.out);
    
    public static void main(String[] args) throws IOException {
        long startTime = System.nanoTime();
        
        int tests = 1;
        // tests = Integer.parseInt(br.readLine());
        
        while (tests-- > 0) {
            solve();
        }
        
        out.flush();
        
        long endTime = System.nanoTime();
        double timeInSeconds = (endTime - startTime) / 1e9;
        long memoryUsed = (Runtime.getRuntime().totalMemory() - Runtime.getRuntime().freeMemory()) / (1024 * 1024);
        
        System.err.println("\n-------------------");
        System.err.printf("Time: %.6f sec\n", timeInSeconds);
        System.err.printf("Memory: %d MB\n", memoryUsed);
    }
    
    static void solve() throws IOException {
        // Your solution here
        
    }
}
